src/constants/cookieNameKey.js

```diff
@@ -13,3 +13,4 @@ governing permissions and limitations under the License.
 export const IDENTITY = "identity";
 export const CONSENT = "consent";
 export const CLUSTER = "cluster";
+export const MBOX = "mbox";
```

Comment:
Is this used or going to be used anywhere in the code, other that tests?

Reply:
reverted it, and using the other one used in the code.